webapps/ui/cockpit/plugins/base/app/views/processInstance/job-retry-bulk-dialog.html

```diff
@@ -118,7 +183,7 @@ <h3>{{ 'PLUGIN_JOB_RETRY_BULK_INCREMENT_RETRIES' | translate }}</h3>
 
   <button class="btn btn-primary"
           ng-click="retryFailedJobs(selectedFailedJobIds)"
-          ng-disabled="!failedJobs || !failedJobs.length || !selectedFailedJobIds.length"
+          ng-disabled="!failedJobs || !failedJobs.length || !selectedFailedJobIds.length || (radio.value === 'dueDate' && (!dueDate || dueDate && !checkDateFormat(dueDate)))"
```

Comment:
🔧 Quite long expressions. It would be nicer to move these into a separate method. There is also one in the dialog html.